wpilibc/src/main/native/include/frc/DriverStation.h

```diff
@@ -266,9 +266,33 @@ class DriverStation : public ErrorBase {
   bool IsBrownedOut() const;
 
   std::string GetGameSpecificMessage() const;
+  /**
+   * Returns the name of the competition event from the FMS.
+   *
+   * @return An std::string containing the event name
+   */
   std::string GetEventName() const;
+
+  /**
+   * Return the type of match being played from the FMS.
+   *
+   * @return The match type enum (kNone, kPractice, kQualification,
```

Comment:
Since the return type is part of the function, we don't need all the allowed values directly in the doc comment.

Reply:
This was done to match the documentation for `GetAlliance`
```c++
  /**
   * Return the alliance that the driver station says it is on.
   *
   * This could return kRed or kBlue.
   *
   * @return The Alliance enum (kRed, kBlue or kInvalid)
   */
  Alliance GetAlliance() const;
```
Although I do agree with your reason for omitting it.